nservicebus/messaging/databus/index.md

```diff
@@ -56,6 +57,20 @@ snippet: DefineMessageWithLargePayloadUsingConvention
 snippet: MessageWithLargePayloadUsingConvention
 
 
+## Serialization
+
+By default NServiceBus uses a `BinaryFormatter` to serialize and deserialize DataBus properties.
+
+
+### Using a custom serializer
+
+To override the DataBus property serializer, create a class that implements `IDataBusSerializer` and add it to the NServiceBus [container](/nservicebus/containers/).
+
+NOTE: Implementations of `IDataBusSerializer.Serialize(...)` must not close the `Stream` instance that is passed in. Implementations of `IDataBusSerializer.Deserialize(...)` may close the `Stream` instance that is passed in, but it is not required.
+
+WARNING: Both the sender and the receiver must use the same DataBus serializer.
```

Comment:
Well, technically it is enough to use _compatible_ serializers i.e. if the receiving serializer feature set can be a superset of sender's serializer features.